pyaedt/hfss3dlayout.py

```diff
@@ -1775,16 +1803,16 @@ def load_diff_pairs_from_file(self, filename):
             return False
         return True
 
-    @pyaedt_function_handler()
-    def save_diff_pairs_to_file(self, filename):
+    @pyaedt_function_handler(filename="input_file")
+    def save_diff_pairs_to_file(self, input_file):
         # type: (str) -> bool
         """Save differtential pairs definition to a file.
 
         If a file with the specified name already exists, it is overwritten.
```

Comment:
Same comment as earlier about possibly moving this to the description for the  ``input_file`` parameter.